app/lib/surface/Surface.coffee

```diff
@@ -100,6 +100,8 @@ module.exports = Surface = class Surface extends CocoClass
     })
     @realTimeInputEvents = @gameUIState.get('realTimeInputEvents')
     @listenTo(@gameUIState, 'sprite:mouse-down', @onSpriteMouseDown)
+    if @world.trackMouseMove
```

Comment:
Could we add a comment or set this to a default value at `world` construction to make it easier to trace?

We have in the past sometimes wondered where a property is magically appearing from. This could be set to false in the constructor with a comment saying that it is mutated by the game codebase.

Reply:
Most of the properties are from Systems in Yeti code, so we don't have initialization for them in the client code (and we don't need because we define default values in Systems parameters).
Agreed as for commenting this. Will add